src/components/Card/Card.module.css

```
@@ -0,0 +1,63 @@
+
+.card {
+    width: 100%;  
+    border: 1px solid #ccc;
+    background-color: #fff;
+    text-decoration: none;
+    color: #000;
+    display:block;
+}
+.card:hover{
+    box-shadow: 0 4px 8px rgba(0, 0, 0.2);
+}
+.card:active{
+    box-shadow: none;
+}
+  
+.title {
+    font-size: 1.25rem;
+    word-wrap: break-word;
+    margin: 0 auto;
+    padding: auto 36px;
+}
+
+.img{
+    position: relative;
+    width: 100%; 
+    aspect-ratio: 360 / 211; 
+
+}
+.text{
+    font-size: 0.75rem;
+    word-wrap: break-word;
+    margin-top: 0;
+    margin-bottom:0;
+    padding-left:36px;
+    padding-right:36px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+.date{
+    font-size: 1rem;
+    margin-right:8px;
+    margin-top:8px;
+    margin-bottom: 0;
+    text-align: right;
+}
+.box {
+    width: 160px; 
+    height: 32px; 
+    /*position: absolute;
+    right: 0;
+    bottom:0;
+    float: right;*/
```

```suggestion
    position: absolute;
    right: 0;
    bottom:0;
    float: right;
```

動かないならとりあえずコメントアウト戻してもらうか